src/nmodl/kinetic.cpp

```diff
@@ -426,6 +426,11 @@ void massagekinetic(Item* q1, Item* q2, Item* q3, Item* q4) /*KINETIC NAME stmtl
             q = q->next;
             qend = ITM(q);
             for (q1 = qb->next; q1 != qend; q1 = q1->next) {
+                // if a state compartment variable is not used in
+                // the kinetic block then skip it
+                if (!SYM(q1)->used) {
+                    continue;
```

Comment:
a bit puzzled why this change ends up reducing coverage in RxD species .. https://app.codecov.io/gh/neuronsimulator/nrn/pull/2210

Reply:
this change is for nocmodl and the cpp file it generates. Not sure how rxd test coverage is shown there. (but I don't see anything obvious have to worry about this change)

Reply:
@alexsavulescu : is it expected that code coverage is a mandatory check and it blocks the merge?

Reply:
@pramodk it does not. Open conversations on the other hand do (n.r. this one :D )

Reply:
🙈